contoso_supermarket/operations/freezer_monitoring/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: sensor-monitor
+description: A helm chart to deploy the Freezer Monitoring application
+home: https://github.com/microsoft/jumpstart-agora-apps
+type: application
+version: 0.1.0
+sources:
+  - https://github.com/charris-msft/jumpstart-agora-apps
```

Comment:
we can remove the sources property or replace your GitHub Id with microsoft/jumpstart-agora-apps

Reply:
Removing sources - that will be more consistent with contoso_supermarket